### PR TITLE
Respond Modal works and a few little changes

### PR DESCRIPTION
**Pull request by YheCoder**
Respond Modal works and a few little changes

### CODE REVIEW

**Comment by hstoebel** [comment]
Hey @YheCoder this looks great! I cloned your repo and I wasn't able to submit a response using the modal. I also wasn't able to submit and up or down vote. Could you show me in class on Friday?

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -104,9 +104,30 @@ def destroy
...
+#    @question.down_vote = @question.down_vote + 1
+#  end
+
+
+  def up_vote
+    @question = Question.find(params[:id])
+    @question.liked_by current_user
+
+    if request.xhr?
+      render json: { count: @question.get_likes.size, id: params[:id] }
```
Comment (hstoebel) [inline app/controllers/questions_controller.rb:122]: where are the methods `liked_by` and `get_likes` defined? I couldn't see them in the `Question` model.

File: app/assets/javascripts/questions.js.coffee [hunk 2]
```diff
@@ -2,29 +2,7 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
-$('#testBtn').click ->
```
Comment (jstoebel) [inline app/assets/javascripts/questions.js.coffee]: I'm a little lost on what these functions do (could you help me understand?)

File: app/assets/javascripts/questions.js.coffee [hunk 3]
```diff
@@ -2,29 +2,7 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (hstoebel) [inline app/assets/javascripts/questions.js.coffee:4]: I'm a little lost on what these two functions do. Could you help me understand?

File: app/controllers/questions_controller.rb [hunk 4]
```diff
@@ -104,9 +104,30 @@ def destroy
...
+    else
+      redirect_to @question
```
Comment (hstoebel) [inline app/controllers/questions_controller.rb:124]: this isn't a valid path.

File: app/controllers/questions_controller.rb [hunk 5]
```diff
@@ -104,9 +104,30 @@ def destroy
     flash[:success] = "You have successfully deleted the question"    
   end
 
+#  def up_vote
```
Comment (jstoebel) [inline app/controllers/questions_controller.rb]: removed commented out method

**Pull request closed without merging**